gui/packages/ubuntupro/lib/l10n/app_en.arb

```diff
@@ -69,18 +69,14 @@
             }
         }
     },
-    "landscapeQuickSetupSaas": "Landscape SaaS configuration",
-    "landscapeQuickSetupSaasHint": "Register with landscape.canonical.com",
-    "landscapeQuickSetupSelfHosted": "Self-hosted Landscape configuration",
-    "landscapeQuickSetupSelfHostedHint": "Register with your own Landscape server",
-    "landscapeFQDNLabel": "Landscape server address",
+    "landscapeSetupManual": "Manual configuration",
+    "landscapeSetupManualHint": "Register with your own Landscape server",
+    "landscapeSetupCustom": "Advanced Configuration",
+    "landscapeSetupCustomHint": "Load a custom Landscape client configuration file",
+    "landscapeFQDNLabel": "Landscape FQDN",
     "landscapeFQDNError": "Invalid URI. Format should be a hostname or IP address.",
```

Comment:
I think handling `landscape.canonical.com` deserves its own error message, don't you think so?

![image](https://github.com/user-attachments/assets/5f39719a-9ad7-4e55-9e42-f237b319ef44)